### DISCUSSION TITLE
Performance question - is it better to use actions if you have a lot of fields

### DISCUSSION THREAD

**Discussion by l-mtc** [filamentphp/filament#10209] [Help]
When creating I have a lot of form fields, and it seems to take a while to load, is it better to break these up into actions? Any other suggestions?

**Comment by clinvest** [comment]
Hi, I believe we need more context. What is a lot of form fields? you can also break them into a wizard form.

Also, my own project is slower on local but pretty fast on my host serv with caching and server correctly configured (OPcache
 etc.)

**Reply by l-mtc** [reply]
It's for a form builder so can be quite a few fields and options, yea I was thinking wizard but in this case it wouldn't make sense. Ah ok, maybe I don't need to worry about it for now then

**Reply by l-mtc** [reply]
@clinvest I've made a fair amount of updates, but ended up getting into a situation where I'm hitting PHP memory limits, I'm trying to reduce the number of components but do you have any suggestions? Currently I'm mostly using repeaters and builder. This is for a page builder so there's quite a few conditional fields

**Reply by ronylicha** [reply]
How many field do you have?
Do you have imported lots of icons set?
